readconf.c

```diff
@@ -1734,18 +1738,140 @@ read_config_file_depth(const char *filename, struct passwd *pw,
 
 	if ((f = fopen(filename, "r")) == NULL)
 		return 0;
+	if (flags & SSHCONF_CHECKPERM) {		
+#if WINDOWS
```

Comment:
I thought about the reuse, but did not come out with a good way:
1. The owner_sid and acl come from the same function call GetSecurityInfo. If split, need to call it twice
2. There are tons of type of known account (around 170) users for a file and also random users. To make the function reusable, we need to know each account and its access mask. for each use, we skip check of different types of account. the out parameter could be big array of accounts and a big array of access masks.